backend/lib/config/gardener.js

```diff
@@ -17,6 +17,8 @@ const environmentVariableDefinitions = {
   VUE_APP_VERSION: 'frontend.appVersion',
   SESSION_SECRET: 'sessionSecret', // pragma: whitelist secret
   API_SERVER_URL: 'apiServerUrl',
+  POD_NAME: 'pod.name',
+  POD_NAMESPACE: 'pod.namespace',
```

Comment:
In the end we want to know the namespace of the lease object (which is the same as the pod) and a holder identity. We do not actually care about the pod name and namespace. We just map the values.
That's why I would prefer to have this in the config (as e.g. `github.lease.namespace` and `github.lease.holderIdentity`) so that when running it locally you do not have to set the env variables. Or is there a reason why this should only be set via env variables?

Reply:
But it is not something what we configure from my point of view. Because the configuration sheould be the same for all pods. And if we add the podNmae to the config this would not be the case.

Reply:
The holder identity is related to the pod instance and should not be in the configuration. We could have the info where the holder identity comes from in the config.